mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
```

Comment:
I like the added strictness, but I wonder if this file is required. I see there is a `mypy --strict` flag. Can that be used instead?

Reply:
Mypy picks automatically this file which means that it works without being invoked via `nox`. This means editors which integrate with `mypy` get the correct errors based on the settings here.

Reply:
Sounds good, I agree having the .ini is better